src/librbd/managed_lock/BreakRequest.h

```diff
@@ -55,9 +61,13 @@ class BreakRequest {
    * @endvertbatim
    */
 
-  ImageCtxT &m_image_ctx;
+  librados::IoCtx &m_ioctx;
+  CephContext *m_cct;
+  ContextWQ *m_work_queue;
+  std::string m_oid;
   const Locker &m_locker;
```

Comment:
@dillaman Don't you think it would be safer to copy the locker in constructor? I just stepped on this in my LeaderWatcher, where I was calling break lock asynchronously and reset  the locker not waiting for the request completed. No problem to change my code, but this was unexpected to me -- I had impression (may be wrong) we had a rule to copy input params like this.